frontend/src/pages/Assessments/data/reducer.ts

```diff
@@ -24,16 +26,30 @@ const assessmentReducer = (
           currentIndex: payload.assessment.currentIndex,
           words: payload.assessment.words,
           firebaseId: payload.assessment.firebaseId,
+          sessionId: payload.assessment.sessionId,
         },
         finished: false,
       };
     case ActionTypes.UPDATE_ASSESSMENT_SUCCESS:
       let { isFinished }: UpdateAssessmentSuccess = payload;
       return {
-        ...state,
```

Comment:
why was this removed?

Reply:
If you are updating the assessment, you should not need the error or the current assessment id again, so we removed it.